hw_4-Component/script.js

```diff
@@ -0,0 +1,131 @@
+
+async function getApi (){
+    try{
+        let response = await fetch('./data.json');
+        let data = await response.json();
+        createContent(data)
+        console.log(data)
+    } catch(err){
+        console.log(err)
+    }finally{
+        console.log('Hy')
+    }
+};
+getApi()
+
+
+function createContent(data){
+    let count 
+    if(localStorage.getItem('notificationId') !==null){
+        count = localStorage.getItem('notificationId')
+    }
+    else{
+        count = 0
+    }
+    let content = document.createElement('div');
+    content.innerHTML = `
+        <div class="notification">
+        <div class="block-content">
+            <div class="content" id="${data[count].id}">
+                <h3 class="contentTitle">${data[count].title}</h3>
+                <p class="content-discription">${data[count].phrase}</p>
+            </div>
+            <img class="close" tabindex="1" src="https://cdn1.iconfinder.com/data/icons/flat-web-browser/100/close-512.png">
```

Comment:
making this a real `<button>` element will not need `tabindex` and event listener on key up to track enter presses - this would be handled in browser by default.
Buttons should be real buttons, this touches all other buttons in your layout.